pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -18,22 +16,22 @@ type REST struct {
 
 // NewStorage returns a RESTStorage object that will work against ClusterPolicy.
 func NewStorage(optsGetter restoptions.Getter) (*REST, error) {
-
 	store := &registry.Store{
-		NewFunc:           func() runtime.Object { return &authorizationapi.ClusterPolicy{} },
-		NewListFunc:       func() runtime.Object { return &authorizationapi.ClusterPolicyList{} },
-		QualifiedResource: authorizationapi.Resource("clusterpolicies"),
+		NewFunc:     func() runtime.Object { return &authorizationapi.ClusterPolicy{} },
+		NewListFunc: func() runtime.Object { return &authorizationapi.ClusterPolicyList{} },
 		ObjectNameFunc: func(obj runtime.Object) (string, error) {
 			return obj.(*authorizationapi.ClusterPolicy).Name, nil
 		},
-		PredicateFunc: func(label labels.Selector, field fields.Selector) storage.SelectionPredicate {
-			return clusterpolicy.Matcher(label, field)
-		},
+		QualifiedResource: authorizationapi.Resource("clusterpolicies"),
```

Comment:
I really need to write that etcd test to make sure we do not accidentally move stuff.